src/cryptoadvance/specter/services/swan/config.py

```diff
@@ -1,6 +1,14 @@
 class BaseConfig:
+    SWAN_CLIENT_ID = "specter-dev"
+    SWAN_CLIENT_SECRET = (
+        "BcetcVcmueWf5P3UPJnHhCBMQ49p38fhzYwM7t3DJGzsXSjm89dDR5URE46SY69j"
+    )
     SWAN_API_URL = "https://dev-api.swanbitcoin.com"
 
 
 class ProductionConfig(BaseConfig):
+    SWAN_CLIENT_ID = "specter"
+    SWAN_CLIENT_SECRET = (
```

Comment:
We should make a remark here for semi-educated security-researchers. Like:
```suggestion
    # Conceptionally, we're using PKCE which should not make any password necessary. However, technically, it
    # was too much challenge to avoid the password and so we technically this password is used but it's not 
    # considered to be a real secret. There is no reasonable attack-vector for this secret being public.
    # Obviously, you're happy to prove wrong but please don't complain about hard-coded secrets or so.
    SWAN_CLIENT_SECRET = (
```

Reply:
Good call. Comment added at the top of the file.